src/specBuilder/bar/stackedBarUtils.ts

```diff
@@ -71,14 +71,15 @@ export const getStackedBackgroundBar = (props: BarSpecProps): RectMark => {
 		name: `${name}_background`,
 		type: 'rect',
 		from: { data: isDodgedAndStacked(props) ? `${name}_facet` : getBaseDataSourceName(props) },
-		interactive: false,
+		interactive: getInteractive(props.children, props),
```

Comment:
```suggestion
		interactive: false,
```

Reply:
Background bars should always be `interactive = false`. The purpose of a background bar is so that when the opacity of a bar lowers, it looks like it is just lightening (or darkening in dark mode). We put a bar directly behind the bar you interact with.